.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
```

Comment:
How does this Travis file fit into the overall deployment of the service?  Can't find it referenced anywhere else in the code.  Does JSL use it?  Do any of our other services use Travis CI?

Reply:
Travis is for [running tests](https://travis-ci.org/mintel/elasticsearch-asg).

Docker images are [built](https://cloud.docker.com/u/mintel/repository/docker/mintel/elasticsearch-snapshooter) [by](https://cloud.docker.com/u/mintel/repository/docker/mintel/elasticsearch-lifecycler) [Docker](https://cloud.docker.com/u/mintel/repository/docker/mintel/elasticsearch-cloudwatcher) [Hub](https://cloud.docker.com/u/mintel/repository/docker/mintel/elasticsearch-healthcheck).

There is zero Jenkins involvement with this repository.

Reply:
I guess my question is more "What triggers the running of Travis within this deployment's infrastructure"?  Where is the code that looks at this configuration file?

Reply:
jtdoepke gave me a quick walk-through; the short answer is that Travis integrates with Github to run and monitor testing.